contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,1020 @@
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Visualize where users click in your app or website.</td>
+        </tr>
+        <tr>
+            <td><strong>Product analytics</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Track events and conversion; analyze user behavior.</td>
+        </tr>
+        <tr>
+            <td><strong>Feature flags</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Roll out features safely; toggle features for cohorts or individuals.</td>
+        </tr>
+        <tr>
+            <td><strong>A/B testing</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Test changes and analyze impact.</td>
+        </tr>
+        <tr>
+            <td><strong>Feedback widgets</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Suggestion box on your site where users can express frustration or delight.</td>
+        </tr>
+        <tr>
+            <td><strong>In-app prompts and messages</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Send messages to users in your app.</td>
+        </tr>
+        <tr>
+            <td><strong>Surveys</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Capture user feedback with surveys. </td>
+        </tr>
+        <tr>
+            <td><strong>Customer interview scheduling</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>In-app widget for requesting customer interviews.</td>
+        </tr>
+        <tr>
+            <td><strong>User interview recordings and transcriptions</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td></td>
```

Comment:
Missing a note or description here (and in all subsequent mentions of it)

Reply:
Hmm Im not sure this needs a note